cluster/gce/configure-vm.sh

```diff
@@ -603,35 +618,33 @@ EOF
   fi
 }
 
+function split-commas() {
+  echo $1 | tr "," "\n"
+}
+
 function try-download-release() {
   # TODO(zmerlynn): Now we REALLy have no excuse not to do the reboot
   # optimization.
 
-  # TODO(zmerlynn): This may not be set yet by everyone (GKE).
   if [[ -z "${SERVER_BINARY_TAR_HASH:-}" ]]; then
     echo "Downloading binary release sha1 (not found in env)"
-    download-or-bust "${SERVER_BINARY_TAR_URL}.sha1"
+    download-or-bust "" "${SERVER_BINARY_TAR_URL}.sha1"
```

Comment:
Good catch. Working on a fix and testing with suppressed hashes (which is what `gke-ci` looks like) now.